### PR TITLE
Fixes Manifest and makes A/G static.

### PR DESCRIPTION
**Pull request by paullewis**
PTAL @brendankenny

### CODE REVIEW

File: audits/manifest/icons-192.js [hunk 1]
```diff
@@ -30,7 +28,7 @@ class ManifestIcons192 {
 
   static audit(inputs) {
     let hasIcons = false;
-    const manifest = manifestParser(inputs.manifest).value;
+    const manifest = inputs.manifest;
```
Comment (brendankenny) [inline audits/manifest/icons-192.js]: you're going to want the uglier check I suggested here. e.g. if they have one icon for several sizes and set `"sizes": "96x96 192x192 256x256"` this check will count that as invalid

File: gatherers/manifest.js [hunk 2]
```diff
@@ -71,7 +72,11 @@ class Manifest extends Gather {
           })
           .then(manifestURL => Manifest._loadFromURL(options, manifestURL));
         })
-        .then(manifest => ({manifest}));
+        .then(manifestContent => {
+          return {
+            manifest: manifestParser(manifestContent).value
```
Comment (brendankenny) [inline gatherers/manifest.js]: hmm, you may want to pass in just `manifestParser(manifestContent)` and make the auditors do the annoying `.value` dance. For manifest exists, at least, an undefined value means no manifest unless `warning` is set, in which case the JSON was invalid

Comment (paullewis) [inline gatherers/manifest.js]: In the current version both equate to a failed audit. I think we should move to "nuanced" fairly soon, especially as the perf numbers start to come in.

File: audits/manifest/icons-192.js (hunk 1 above)
Comment (paullewis) [inline audits/manifest/icons-192.js]: Done.

**Comment by paullewis** [comment]
LGTY?

File: gatherers/manifest.js (hunk 2 above)
Comment (brendankenny) [inline gatherers/manifest.js]: yeah, I was just writing the same, since we don't distinguish between missing/erroneous values at this point

**Comment by brendankenny** [comment]
:microscope: LGTM 
:fireworks:

**Pull request merged**